pkgs/applications/misc/blender/wrapper.nix

```diff
@@ -2,17 +2,15 @@
 , lib
 , blender
 , makeWrapper
-, python3Packages
-}:
-{ name ? "wrapped"
-, packages ? []
+, extraModules ? []
 }:
+{ name ? "wrapped" }:
```

Comment:
Is this actually useful? I feel like moving it to `overrideAttrs` is fine, if people really really want this behavior for some reason.

Reply:
I have no idea. @lucasew wrote that line, and I may need a sec to figure out how to migrate this to work with overrideAttrs. 

I don't know a way to override `withPackages` directly, but it probably will require overriding blender's `buildEnv` that's declared in `default.nix`, so it'd be something like `(blender.override { wrapperName = "withBpycv" }).withPackages ( ... )` or something

If we remove that outright it will be a breaking change

Reply:
@K900 I don't know how github comments work, do people preferred being pinged? Should I ping you?

Reply:
You get emails when people reply to your thread, so no need for a ping. You can basically achieve this (I think?) with `blender.overrideAttrs(old: { name = "blender-fancy-${old.version}"; })`, which can be done in the `blender-with-packages` shim.

Reply:
> Is this actually useful? I feel like moving it to `overrideAttrs` is fine, if people really really want this behavior for some reason.

It's for the suffix of some stuff like desktop items so when installing it differentiate between each wrapper generated. As far as I remember.

Reply:
I'm going to sweep everything under `pname`

Reply:
> > Is this actually useful? I feel like moving it to `overrideAttrs` is fine, if people really really want this behavior for some reason.
> 
> It's for the suffix of some stuff like desktop items so when installing it differentiate between each wrapper generated. As far as I remember.

Wait a minute... could this mean that multiple version of blender could theoretically be installed with this wrapper?

This... could change things

(I should probably stay within scope for now though)

Reply:
> You get emails when people reply to your thread, so no need for a ping. You can basically achieve this (I think?) with `blender.overrideAttrs(old: { name = "blender-fancy-${old.version}"; })`, which can be done in the `blender-with-packages` shim.

now its `(blender.withPackages ( ... )).overrideAttrs ({ pname = "blender-whatever"; })`